frontstage/controllers/iac_controller.py

```diff
@@ -21,18 +21,16 @@ def get_iac_from_enrolment(enrolment_code, validate=False):
     except requests.exceptions.HTTPError:
         if response.status_code == 404:
             logger.info('IAC not found')
-            if validate:
-                raise ApiError(logger, response, message='Invalid enrolment code used')
             return
```

Comment:
Theres no `return` here, you dont need this, or you need a `pass`/`continue`?

Reply:
I don't really get what you mean there is a return right there.

Reply:
i mean you're better off using a `pass`

Reply:
You want to `return` early instead of returning the `response.json()` at the end of the function.

Reply:
i think i've got my python mixed up a little, just seems weird you're returning nothing here

Reply:
Yep yep. This `return` is useful because it says "I don't want this function to do anything more".